src/future-hopr-lib-components/Button/gray.tsx

```diff
@@ -26,14 +25,11 @@ const SButton = styled(MuiButton)`
     background-color: #dfdfdf;
     color: #414141;
   }
-
-  &:focus {
-    background: rgba(255, 255, 255, 0.75);
-    color: #000;
-  }
 `;
 
-export default function Button(props) {
+type ButtonProps = MuiButtonProps;
```

Comment:
if you do not extend it, its not needed

Reply:
I did it so we could have:
```ts
type ButtonProps = MuiButtonProps & {
    our props: type of our props;    
};
```
Should I add this change here?